Import WPT css/compositing directory

mattwoodrow opened the pull request:
#### 94ce519cd2eaba3a1f2f4cb9753f40c98e9d9472
<pre>
Import WPT css/compositing directory
<a href="https://bugs.webkit.org/show_bug.cgi?id=261129">https://bugs.webkit.org/show_bug.cgi?id=261129</a>
&lt;rdar://problem/114954604&gt;

Reviewed by Tim Nguyen.

web-platform-tests revision b6d002e21b44f4afd75bf1d959c65a1e5ce3c927.

Canonical link: <a href="https://commits.webkit.org/267654@main">https://commits.webkit.org/267654@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b8df1771f6cb7197bcc8c3940670a24b8cd77d47

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/17295 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/17620 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/18124 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/19083 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/16183 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/20895 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/17763 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/18371 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/17499 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/17839 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/15037 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/19900 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/15079 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/15727 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/22406 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/16078 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/15895 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/20230 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/16478 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/13985 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/15625 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/15567 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4132 "Built successfully and passed tests") | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/19995 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/16313 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/compositing/Blending_in_a_group_with_filter-ref.html:
Can we fix the tests with -ref.html files but don't link them via `<link rel="match">` upstream ? This way, we can use reftest comparison instead of render tree dump comparison.

mattwoodrow commented on LayoutTests/imported/w3c/web-platform-tests/css/compositing/Blending_in_a_group_with_filter-ref.html:
These tests seem pretty broken, they're not running on wpt.fyi for the same reason :(

I've removed them from this import for now, and will file an issue against WPT.

mattwoodrow commented on LayoutTests/imported/w3c/web-platform-tests/css/compositing/Blending_in_a_group_with_filter-ref.html:
https://github.com/web-platform-tests/wpt/issues/41828

mattwoodrow committed:
Import WPT css/compositing directory
https://bugs.webkit.org/show_bug.cgi?id=261129
<rdar://problem/114954604>

Reviewed by Tim Nguyen.

web-platform-tests revision b6d002e21b44f4afd75bf1d959c65a1e5ce3c927.

Canonical link: https://commits.webkit.org/267654@main
